lib/matplotlib/textpath.py

```diff
@@ -385,11 +385,11 @@ def __init__(self, xy, s, size=None, prop=None,
 
         self._cached_vertices = None
         s, ismath = Text(usetex=usetex)._preprocess_math(s)
-        self._vertices, self._codes = text_to_path.get_text_path(
-            prop, s, ismath=ismath)
+        super().__init__(
```

Comment:
perhaps rather use fast_from_codes_and_verts for speed?

Reply:
That's a class method / constructor, though?

Reply:
ah yes.  Oh well, I guess this isn't a bottleneck...